countdown.js

```diff
@@ -0,0 +1,5 @@
+let x=10;
+while (x > 0) {
+  console.log(x);
+  x--;
+}
```

Comment:
Make sure to leave a space after the variable declaration and a space around the = sign

```
let x = 10;

while (x > 0) {
  console.log(x);
  x--;
}
```